fix/docker-network-doc (#384)
kestra-io/kestra.io: content/docs/01.getting-started.md

Open [http://localhost:8080](http://localhost:8080) in your browser.

> ### Some warnings regarding this docker compose

Reviewer: ```suggestion
### Networking in this docker compose
```

People will skip this part if they don't interesting in networking so the content can be simplified

> The above docker-compose file doesn't specify anything regarding network of the Kestra's service.

Reviewer: ```suggestion
The above docker-compose file doesn't specify anything regarding the network of Kestra's service.
```

> This means that as it is, you won't be able to access any of your localhost exposed services (eg. another docker container with mapped port) even if you're able to reach it from your machine.

Reviewer: ```suggestion
This means that as it is, you won't be able to access any of your localhost exposed services (eg. another docker container with a mapped port) even if you're able to reach it from your machine.
```

> If you don't plan to use local services and reach them from Kestra, skip this part.
> 
> If you do, you have 4 options:

Reviewer: ```suggestion
If you plan to use local services and reach them from Kestra, you have four options:
```

- Depending on your docker implementation, you should be able to use the `host.docker.internal` hostname or `172.17.0.1` to reach your host machine from inside Kestra's container.

> - By default your container will be in a `default` network. You can simply add your other services in the above `docker-compose.yml` file and use services' alias (keys from `services`) to reach them.

Reviewer: ```suggestion
- By default, your container will be in a `default` network. You can add your other services in the above `docker-compose.yml` file and use the services' alias (keys from `services`) to reach them.
```

Always avoid terms like 'simply' as some people may not find it simple

> - You can create a new network and add your services in it. Then you can add this network to the `networks` section of the `kestra` service. You will then have access to a localhost with all your exposed ports.

Reviewer: ```suggestion
- You can create a new network and add your services to it. Then you can add this network to the `networks` section of the `kestra` service. With this, you will have access via `localhost` to all your exposed ports.
```

> - You can use the `host` network mode for the `kestra` service. This will make your container use your host network and you will be able to reach all your exposed ports. (this is the easiest way but can be a security issue)

Reviewer: ```suggestion
- You can use the `host` network mode for the `kestra` service. This will make your container use your host network and you will be able to reach all your exposed ports. This is the easiest way but it can be a security risk.
```
